INSTALL.rst

```diff
@@ -4,24 +4,79 @@ Installing boofuzz
 Prerequisites
 -------------
 
-Boofuzz requires Python. Recommended installation requires ``pip``.
+Boofuzz requires Python 2.7 or ≥3.5. Recommended installation requires ``pip``.
+To ensure forward compatibility, python 3 is recommended. As a base
+requirement, the following packages are needed:
+
+Ubuntu/Debian
+  ``sudo apt-get install python3-pip python3-venv``
+OpenSuse
+  ``sudo zypper install python3-pip python3-virtualenv``
```

Comment:
Is there no python3-venv package for OpenSuse? If not, the venv module might already be integrated in the standard Python installation. Ubuntu and Debian seem to be an exception here, outsourcing venv in a dedicated package.
https://stackoverflow.com/questions/41573587/what-is-the-difference-between-venv-pyvenv-pyenv-virtualenv-virtualenvwrappe

Also, venv is not available for Python 2, so one has to fallback to virtualenv.
The Flask [install docs](https://flask.palletsprojects.com/en/1.1.x/installation/#virtual-environments) handle this quite nicely.

Reply:
I'll install an OpenSuse today and test it out - I haven't used anything Suse based for like 10 years, but I'll check.